.travis/script.sh

```diff
@@ -39,11 +39,21 @@ case "${BUILD_TARGET}" in
         ;;
 
     linux-auto-gcc-check-happy)
+        ls /tmp/happy_debug*
+        mkdir -p $TRAVIS_BUILD_DIR/happy-test-logs/linux-auto-gcc-check-happy/from-tmp
```

Comment:
This block is already switching on BUILD_TARGET, so there's no need to copy-and-paste the literal value of BUILD_TARGET, just reference the variable.